<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetInfoWidget</class>
 <widget class="QWidget" name="SetInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改蓝牙名称</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="pb_Close">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_oldName">
     <property name="text">
      <string>原蓝牙名</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_oleName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_newName">
     <property name="text">
      <string>新蓝牙名</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le_newName"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pb_OK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
